test/HelmetDeclarativeTest.js

```diff
@@ -2501,6 +2501,41 @@ describe("Helmet - Declarative API", () => {
         });
     });
 
+    describe("deferred tags", () => {
+        beforeEach(() => {
+            window.__spy__ = sinon.spy();
+        });
+
+        afterEach(() => {
+            delete window.__spy__;
+        });
+
+        it("executes synchronously when defer={true} and async otherwise", done => {
+            ReactDOM.render(
+                <Helmet>
+                    <script defer={false}>
```

Comment:
`defer` is (I believe) specific to the `<script>` tag. So this solution wouldn't work for style or any other tag. Perhaps it makes more sense to bubble up the attribute to the `<Helmet>` wrapper? That would give devs the flexibility to sync-render the entire Helmet _instance_, rather than the individual tag.

```
<Helmet defer={false}>
```